### ISSUE TITLE
Custom m3u8 for map audio

### ISSUE THREAD

**Issue by rudnypc** [m1k1o/go-transcode#27]
When video has multiple audio, maybe it would be nice if we can choose the best audio language, the ffprobe can give all index audio.

**Comment by rudnypc** [comment]
Generate m3u8 for custom video, audio and subtitle index could be nice too.

**Comment by m1k1o** [comment]
As separate manifest? E.g. `profile-eng.m3u8` or joined together in one playlist as multiple audios?

**Comment by rudnypc** [comment]
It's a nice idea, but how it will work if the video has multiple audios in english or unknow language? What do you think in profile-eng-00.m3u8, profile-eng-01.m3u8 and, profile-ger-00.m3u8, profile-por-00.m3u8, profile-UNK-00.m3u8, profile-UNK-02.m3u8?

unk=unknow

**Comment by m1k1o** [comment]
Yes, I agree, stream index could be used. And then maybe either using filter options as query parameters or in the name of manifest.

**Comment by rudnypc** [comment]
profile-0.m3u8
profile-1.m3u8
profile-2.m3u8

Inside the master playlist, you could put the language name/details.
